### ISSUE TITLE
[데이터셋검토] MuSiQue-Ans

### ISSUE THREAD

**Issue by aigovsensing** [aigovsensing/dataset-review#83]
### 데이터셋 명칭

MuSiQue

### 논문 주소 (URL)

https://arxiv.org/abs/2108.00573

### 데이터셋 저장소 URL

https://huggingface.co/datasets/dgslibisey/MuSiQue

### 추가 참고 사항

MuSiQue-Ans is a new multihop QA dataset with ~25K 2-4 hop questions using seed questions from 5 existing single-hop datasets.

Usage Caution: If you're using any of our seed single-hop datasets (SQuAD, T-REx, Natural Questions, MLQA, Zero Shot RE) in any way (e.g., pretraining on them), please note that MuSiQue was created by composing questions from these seed datasets. Therefore, single-hop questions used in MuSiQue's dev/test sets may occur in the training sets of these seed datasets. To help avoid information leakage, we are releasing the IDs of single-hop questions that are used in MuSiQue dev/test sets. Once you download the data below, these IDs and corresponding questions will be in data/dev_test_singlehop_questions_v1.0.json. If you use our seed single-hop datasets in any way in your model, please be sure to avoid using any single-hop question IDs present in this file
